src/show_up.md

```diff
@@ -12,6 +12,7 @@ At the same time:
 
 * If you are going to raise an objection, you have to be willing to make your case. You can't raise concerns and block indefinitely.
 * It may well be that others *do* understand your point, but just don't agree about how important it is. Ultimately, you have to [trust] that others have the best interest of Rust at heart, too.
+* The hardest disagreements to resolve often have underlying disagreements in values or the priorities of those values. Resolving such disagreements in an ideal way often requires introspecting on your own values, and understanding and empathizing with the values of others.
```

Comment:
This reminds me of something. I was thinking about how it'd be nice to talk about how the best folks find ways to "lift up" the conversation, i.e., looking for cases where it seems like the disagreement is about *values* and opportunities to make those values explicit so that you can find other ways to honor them.

Reply:
Absolutely. You can disagree with someone's values are priorities but still successfully frame something in terms of those values and priorities by way of understanding them and operating in their value system.